src/renderer/file-manager.ts

```diff
@@ -68,7 +69,39 @@ export class FileManager {
     if (!filePath || typeof filePath !== 'string') return;
 
     const editorValues = {};
-    for (const [name, value] of Object.entries(await readFiddle(filePath))) {
+    const files: [string, string][] = Object.entries(
+      await readFiddle(filePath),
+    );
+    for (const [name, value] of files) {
+      if (name === PACKAGE_NAME) {
+        const { remoteLoader } = window.ElectronFiddle.app;
+        const { dependencies, devDependencies } = JSON.parse(value);
+        const deps: Record<string, string> = {
+          ...dependencies,
+          ...devDependencies,
+        };
+
+        if (deps.electron) {
+          // Strip off semver range prefixes.
```

Comment:
The code is probably fine, but when I'm reading through this first time, I hit this paragraph and didn't understand it after five seconds, so I'm going to say the comment is not doing its job of explaining what's going on. :smile: 

Please change it to something like

```js
// Strip off semver range prefixes, e.g.
// before -> after
```

Where `before -> after` is a demo of what to expect